pkg/apis/cluster/v1alpha1/placement_types.go

```diff
@@ -0,0 +1,48 @@
+package v1alpha1
```

Comment:
Are these defined in a shared library that we could just import instead of redefining it for this controller?

Reply:
Yes, but hit issues mentioned above when trying to use it.